core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -278,7 +278,9 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/content_fields/EditorHtmlFieldDirective.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/content_fields/EditorPlaintextFieldDirective.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/simple_editor_tab/questions/TextInteractionDirective.js"></script>
```

Comment:
Perhaps TextEditorDirective.js, for parity with MultipleChoiceEditorDirective.js.

Reply:
Done.